core/src/wsv.rs

```diff
@@ -459,12 +461,8 @@ impl WorldStateView {
     }
 
     /// Returns iterator over blockchain blocks starting with the block of the given `height`
```

Comment:
Docs require update

Reply:
A `std::vec::Vec` _is_ an iterator.

Reply:
Yeah, but we are no longer return blocks, but blocks hashes.